testsuite/pyproject.toml

```diff
@@ -0,0 +1,56 @@
+[build-system]
+# Minimum requirements for the build system to execute
+requires = [
+    "packaging",
+    "setuptools",
+    "wheel",
+]
+
+[project]
+name = "MDAnalysisTests"
+version = "2.1.0-dev0"
```

Comment:
I think we can avoid updating this by using `dynamic = ['version', 'description']`, or similar, based on me cheating and looking at other projects.